README.rst

```diff
@@ -90,6 +124,15 @@ file within that directory contains a refresh token for the user who authorized
 themselves. This refresh token should be treated as if it were a password and
 not shared or otherwise disclosed!
 
+Examples:
+ - ``courseraprogramming upload $MY_CONTAINER_IMAGE $COURSE_ID $ITEM_ID
```

Comment:
This will change with the introduction of branches but looks fine for now.

We'll to find a way to expose the BRANCH_ID to the instructors and will need to replace COURSE_ID with BRANCH_ID. This command should continue working for courses that do not use branches since the branch_id = course_id for the default/first branch of the coure.